packages/tradershub/src/modals/AddOrManageAccount/CurrenciesForm.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { twMerge } from 'tailwind-merge';
+import { TCurrencyConfig } from '@/hooks/useCurrencies';
+import CurrencyCard from '@/screens/CurrencySelector/CurrencyCard';
+import { Button, useDevice } from '@deriv-com/ui';
+
+type TCurrenciesForm = {
+    currencies: TCurrencyConfig[];
+    submitButtonLabel?: string;
+};
+
+const CurrenciesForm = ({ currencies, submitButtonLabel }: TCurrenciesForm) => {
+    const { isDesktop } = useDevice();
+    return (
+        <Formik
+            initialValues={{
+                currency: '',
+            }}
+            // will add submit handler later with the API call
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
+            onSubmit={() => {}}
+        >
+            {() => (
+                <Form className='flex flex-col items-center justify-between h-full min-h-0 py-16 lg:px-16 lg:p-24'>
+                    <div
+                        className={twMerge(
+                            'overflow-y-auto flex flex-wrap justify-start w-full lg:w-[80%]',
+                            currencies?.length < 4 ? 'lg:justify-center' : ''
+                        )}
+                    >
+                        {currencies.map(currency => (
```

Comment:
@shahzaib-deriv Can you add the logic of added accounts here ? so we can disable the added accounts which are already added.

we need a hook for this to filter the available currencies based on the added currencies.

Reply:
Sure thing @mahdiyeh-deriv 
I reduced the story points for the cards and in this PR we're focusing on screen implementation listing out the currencies and responsive styles.

Reply:
We need to refine the hook to add `info` flag as well. To keep it simple we can implement in a separate PR as a business logic. 
If you prefer we can do this in the same PR as well

Reply:
@shahzaib-deriv To me both are fine, If you prefer to keep it as a separate card I can add it under the same US, but I want to make sure this will be done right after merging this PR and not going to be postponed.

Reply:
Yes @mahdiyeh-deriv this is going to be the immediate implementation after these UI screens